api/schemas.py

```diff
@@ -14,3 +14,16 @@ class Meta:
         self_view = 'books.get_by_id'
         self_view_kwargs = {'book_id': '<id>'}
         self_view_many = 'books.get_all'
+
+
+class AuthorSchema(Schema):
```

Comment:
Prefer the approach of separating classes into different files. It makes source navigation easier.